agent/utils/mobypkgwrapper/plugins.go

```diff
@@ -31,5 +31,7 @@ func NewPlugins() Plugins {
 }
 
 func (*plugins) Scan() ([]string, error) {
-	return mobyplugins.Scan()
+	// Delegate the call to the mobyplugins package
```

Comment:
nit: I'm assuming this change was necessary due to perhaps `Scan` being turned from a standalone function into a method of LocalRegistry upstream. If so, can we call this out in the PR description as well?

Reply:
will add that to the PR description, ty!